dev/readthedocs.md

```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
 
 3. How do I get a terminal running in my docker container?
+   docker-compose up
 
 4. Where do I store data I don’t want to commit?
+   add to .gitignore
 
 5. What is our stack?
+   Our stack includes Database, Backend and Frontend Frameworks, and CSS Framework. 
```

Comment:
More specifically, what are these frameworks?

Reply:
We use PostgreSQL with Django ORM for the database, Django with pycodestyle linter and Black formatter for backend, React/Redux with ESLint linter and Prettier formatter for frontend, and SCSS preprocesor with the BME methodology and Airbnb style guide for CSS.